### ISSUE TITLE
server error 500 when adding end date

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispers#691]
I see that 2.3.0 just got pushed live... I'm having trouble adding an end date to 170439 in the live version. I get this error

Error. Event location details not updated. Error message: {"error":"Server Error (500)"}

**Comment by JChipault** [comment]
Hmmm... I also can't update the Process Status field on this event but I could on other more recent events in live WHISPers. Another NWHC also couldn't update Process Status when they tried it late last month. I don't get a server error for process status, just nothing happens when I hit "save" on the edit event subform. Is there something different about this event?

**Comment by aaronstephenson** [comment]
Hi @JChipault. This is not related to the client version update. The error was caused by trying to save a null value in a non-nullable field, in this case `legal_number`.

TL;DR: I have a really easy and quick fix.

The long version:

I am aware that the business rules doc says that this field should be nullable, but it is also defined as a VarChar type, and in Django those two are not compatible:
https://docs.djangoproject.com/en/2.2/ref/models/fields/#null
>Avoid using null on string-based fields such as CharField and TextField. If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string. In most cases, it’s redundant to have two possible values for “no data;” the Django convention is to use the empty string, not NULL.

So, the way this field is defined in the models.py field is to allow blank (empty string) values, and set the default value to `''` in case the user does not explicitly submit that field. For each new event created in WHISPers 2.0 this all works seamlessly. But I just now noticed that all of your legacy data (of which Event 170439 is a part) have nulls in this field.

So, the solution is to set all nulls in this field to empty string, which I will do right now.

The problem you are encountering with `process_status` is something else entirely; Lauren says it is due to the bug she found in issue #684, and she has a fix for it in the issue branch of this repo (which hasn't been merged into test or prod yet).
